distro/configs/openfn/openfn.env

```diff
@@ -0,0 +1,191 @@
+# Default values are optimized for production to avoid having to configure
+# much in production.
+#
+# However it should be easy to get going in development too. If you see an
+# uncommented option that means it's either mandatory to set or it's being
+# overwritten in development to make your life easier.
+
+# Set this up to handle GitHub App configuration
+# GITHUB_APP_ID=12345
+# GITHUB_CERT=Base64-encoded-private-key
+
+# Set this up to handle SalesForce OAuth credentials
+# SALESFORCE_CLIENT_ID=3MVG9_ghE
+# SALESFORCE_CLIENT_SECRET=703777B
+
+# Set this up to handle Google OAuth credentials (ex: GoogleSheets)
+# GOOGLE_CLIENT_ID=660274980707
+# GOOGLE_CLIENT_SECRET=GOCSPX-ua
+
+# Choose an admin email address and configure a mailer. If you don't specify
+# mailer details the local test adaptor will be used and mail previews can be
+# viewed at localhost:4000/dev/mailbox
+EMAIL_ADMIN='admin@openfn.org'
+# MAILGUN_API_KEY='some-key'
+# MAILGUN_DOMAIN='some-domain'
+
+# You should generate a random string of 64+ characters for this value in prod.
+# You can generate a secure secret by running: ./run secret
+SECRET_KEY_BASE=please_generate_a_more_secure_unique_secret_value_for_your_project
+
+# Which environment is running? MIX_ENV should be "dev" or "prod" and NODE_ENV
+# should be "production" or "development". When MIX_ENV is set to prod you'll
+# automatically be set to build and run releases instead of using mix.
+#MIX_ENV=prod
+#NODE_ENV=production
+MIX_ENV=dev
+NODE_ENV=development
```

Comment:
Do we need to change this in production? or does the build script handle this in someway?

Reply:
I am not good in mix environments.  @taylordowns2000 might have the best answer.

Reply:
yes, both should be set to production. here what our GKE config map looks like:
```
"MIX_ENV": "prod",
"NODE_ENV": "production",
```

Reply:
@taylordowns2000, @aleksa-krolls is there a way to manage OpenFN configs and generate tokens without manually setting them up using the UI + update of openfn > config.json? 

For example, using the .env variables or maybe the Maven scaffolding similarly to how we do for OpenMRS.